### PR TITLE
Add basic main menu

### PR DESCRIPTION
**Pull request by zainafzal08**

### CODE REVIEW

File: src/main-menu.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      height: 100vh;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+      flex-direction: column;
+    }
+  `;
+
+  @property({ type: Boolean })
+  settingsOpen = false;
```
Comment (josephkimsungha) [inline src/main-menu.ts]: nit: this can be a `@state`

File: src/main-menu.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  }
+
+  render() {
+    if (this.settingsOpen) {
+      return this.renderSettings();
+    }
+
+    return html`
+      <h1> Power Lifter </h1>
+      <button @click=${this.startGame}> Play </button>
```
Comment (josephkimsungha) [inline src/main-menu.ts:28]: Does this need arrow notation?

File: src/main-menu.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      align-items: center;
+      justify-content: center;
+      flex-direction: column;
+    }
+  `;
+
+  @property({ type: Boolean })
+  settingsOpen = false;
+
+  startGame() {
```
Comment (josephkimsungha) [inline src/main-menu.ts]: nit: private (here and below)

also can we put the order as
- render
- renderSettings
- startGame

**Review by josephkimsungha — APPROVED** [review]

File: src/main-menu.ts (hunk 2 above)
Comment (zainafzal08) [reply src/main-menu.ts:28]: Nope! 

https://lit.dev/docs/components/events/#adding-event-listeners-in-the-element-template

**Pull request merged**